docs/style.css

```diff
@@ -0,0 +1,557 @@
+:root {
+    --dark: #403F3D;
+    --light:#E1D4C9;
+    --accent:#B0907A;
+    --container:#665F55;
+    --backdrop:#403F3D 80%;
+}
+body {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    max-width: 1440px;
+    max-height: 4388px;
+    justify-content: space-between;
+    text-align: center;
+    align-items: center;
+    background-color: var(--light);
+    color: var(--dark);
+    font-family: 'inter';
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 150%;
+    gap: 20px;
+}
+
+h1 {
+    font-size: 72px;
+    font-weight: 600;
+    line-height: 105%;
+}
+
+h2 {
+    font-size: 60px;
+    font-weight: 600;
+    line-height: 125%;
+}
+
+h3 {
+    font-size: 24px;
+    font-weight: 600;
+    line-height: 125%;
+}
+
+ul {
+    list-style: none;
+    
+}
+
+ul li {
+    display: inline-block;
+    padding: 0 20px;
+}
+
+header {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-content: center;
+    gap: 5px;
+}
+
+main {
+    margin-bottom: 100px;
+}
+
+footer {
+    display: flex;
+    height: 450px;
+    background: var(--container);
+    border-radius: 40px;
+    align-self: flex-end;
+    margin-left: 100px;
+}
+
+a {
+    scroll-behavior: smooth;
+    text-decoration: none;
+    color: var(--dark);
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 150%;
+}
+
+#hero {
+    margin-top: 20px;
+}
+
+#applestore-btn, #playstore-btn {
+    display: flex;
+    align-items: center;
+    background: var(--light);
+    padding: 12px 20px;
+    width: 200px;
+    height: 64px;
+    border: 1px solid var(--dark);
+    border-radius: 100px;
+    gap: 8px;
+}
+
+footer#contacts {
```

Comment:
Over here would be better to add a class name, overwritting the styles could be hard later. Leave it as is, but for the future, try not to select styling by ids.